ChecksFile.wl

```diff
@@ -1,6 +1,6 @@
 (* ::Package:: *)
 
-(* ::Subsection:: *)

```

Comment:
It is very good practice to commit purely formatting changes in a single separate commit like this. Nice!